srcpkgs/olm-python/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'olm-python'
+pkgname=olm-python
+version=3.0.0
+revision=1
+wrksrc="olm-${version}"
+build_wrksrc=python
+build_style=python-module
+pycompile_module="olm"
+hostmakedepends="python-setuptools python3-setuptools python-cffi python3-cffi"
+makedepends="python-devel python3-devel libffi-devel olm-devel"
+depends="python python-cffi python-future python-typing"
+short_desc="Implementation of the Double Ratchet cryptographic ratchet (Python2)"
+maintainer="Adam Beckmeyer <adam_gpg@thebeckmeyers.xyz>"
+license="Apache-2.0"
+homepage="https://git.matrix.org/git/olm/about/"
+distfiles="https://git.matrix.org/git/olm/snapshot/olm-${version}.tar.gz"
+checksum=4c54f93e4f426a9d5c8e66e63113d42fd2afca240cdcff30a469a342a6ec3bc6
+
+pre_build() {
+	make include/olm/olm.h
+}
+
+olm-python3_package() {
+	short_desc="${short_desc/Python2/Python3}"
+	depends="${sourcepkg}>=${version}_${revision} python3 python3-cffi
```

Comment:
ditto

Reply:
Fixed. Was a relic from when it was `noarch`.